irohad/ametsuchi/impl/postgres_block_query.hpp

```diff
@@ -23,25 +23,25 @@
 #include <pqxx/nontransaction>
 
 #include "ametsuchi/block_query.hpp"
-#include "ametsuchi/impl/flat_file/flat_file.hpp"
+#include "ametsuchi/impl/flat_file/impl/flat_file_impl.hpp"
 #include "logger/logger.hpp"
 #include "postgres_wsv_common.hpp"
 
 namespace iroha {
   namespace ametsuchi {
 
-    class FlatFile;
+    class FlatFileImpl;
 
     /**
      * Class which implements BlockQuery with a Postgres backend.
      */
     class PostgresBlockQuery : public BlockQuery {
      public:
       PostgresBlockQuery(pqxx::nontransaction &transaction_,
-                         FlatFile &file_store);
+                         FlatFileImpl &file_store);
```

Comment:
Original idea was to test `PostgresBlockQuery` without concrete implementation as I understand. Using FlatFileImpl here will not allow passing a mock to constructor. So `FlatFile` should be used in this file instead if I am not mistaken.

Reply:
You are right, auto-refactoring changed that, I did not see